src/app/test/db/mongo/util_test.py

```diff
@@ -365,3 +365,15 @@ def test_ensure_roles_creates_superuser_role_if_none_exists(self, monkeypatch):
         superuser = Role.objects.get(name="superuser")
 
         assert len(superuser.permissions) == len(Permissions)
+
+    def test_ensure_roles_creates_bg_plugin_role_if_none_exists(self, monkeypatch):
+        """A plugin role with all permissions should be created if none exists"""
+        monkeypatch.setattr(
+            beer_garden.db.mongo.util, "_sync_roles_from_role_definition_file", Mock()
+        )
+
+        assert len(Role.objects.filter(name="plugin")) == 0
+        ensure_roles()
+        plugin_role = Role.objects.get(name="plugin")
+
+        assert len(plugin_role.permissions) == 8  # Roles defined in util.py
```

Comment:
This check feels a little too fragile.  Anytime we change the permissions that are associated with the role we have to come back and touch this test.  Two options:

* Just check that the role got created and call that good.
* Put the list of permissions associated with the role in a constant that can get imported here.  So you'd have something like:

```python
from beer_garden.db.mongo.util import PLUGIN_ROLE_PERMISSIONS
...
...
    assert plugin_role.permissions == PLUGIN_ROLE_PERMISSIONS
```

Then this test continues to pass no matter how that list changes.